style_editor.html

```diff
@@ -126,87 +279,7 @@
   border: 1px solid #ccc;
   border-top: none;
 }
-/* Do Selected Effect button */
```

Comment:
This was just moved right?
Please try not to mix changes that do something with stuff that do stuff, it makes it take much much longer to figure out what has changed...

Reply:
This is indeed just a move, but it came with the addition of the other rules.
This is leading toward a final order of CSS rules where the cascade matters.